cobalt/dom/media_source.cc

```diff
@@ -439,6 +426,8 @@ bool MediaSource::IsTypeSupported(script::EnvironmentSettings* settings,
 
 bool MediaSource::StartAttachingToMediaElement(
     HTMLMediaElement* media_element) {
+  is_using_media_source_attachment_methods_ = false;
```

Comment:
Just check again if this will overwrite `is_using_media_source_attachment_methods_` even we set it via H5VCC flag?

Reply:
The H5VCC flag will affect any new HTMLMediaElements created after the flag value is updated. Any existing media elements will continue to use the value that they were created with, so it is expected that this may differ from the most recent value of the flag.

This helps ensure that any MediaSource objects interacting with a HTMLMediaElement will use the same value as when the media element was created. Testing without this, I was getting flag mis-matches between the element, media source, and source buffers, which was resulting in crashes.